scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1814,9 +1817,17 @@ then
         DONOR_MYSQL_VERSION=$(parse_sst_info "$sst_file_info_path" sst mysql-version "")
 
         transition_key=$(parse_sst_info "$sst_file_info_path" sst transition-key "")
+
+        # Use the broken key if the donor version is < 5.7.29 and is not 5.7.28-31.31-41.2
```

Comment:
The other direction (fixed donor, old receiver) will be unsupported? Because then that's something we should document.

Reply:
yes, the donor does not know the joiner version (so we don't know what transition key to use).  I have added a comment to the jira ticket.